src/python/espressomd/particle_data.pyx

```diff
@@ -1678,6 +1678,62 @@ cdef class ParticleList:
                     if (p.type == t or t == "all"):
                         vtk.write("{} {} {}\n".format(*p.v))
 
+
+    def writevsf(self, fp, types='all'):
+        """
+        writes a VST (VTF Structure Format) to a file.
+        This can be used to write the header of a VTF file.
+
+        Parameters
+        ----------
+        types : str
+                Specifies the particle types. The string 'all' will write all particles
+        fp : file
+                   File pointer to write to.
+
+        """
+        global box_l
+        if not hasattr(types, '__iter__'):
+            types = [types]
+            
+        fp.write("unitcell {} {} {}\n".format(*(box_l)))
+        for p in self:
+            for t in types:
+                if (p.type == t or t == "all"):
+                    fp.write("atom {} radius 1 name {} type {} \n".format(p.id, p.type, p.type))
```

Comment:
This is a generic value for VMD, the particles don't necessarily have a LJ potential so it is not obvious to find any meaningful value.

If we want to make it something more specific than just 1, perhaps:
- 0.5 since it corresponds  the size of a WCA bead: sigma.
- some radius from the friction coefficient gamma (although I think that will require LANGEVIN_PER_PARTICLE)